disco_aws_automation/disco_acm.py

```diff
@@ -6,27 +6,45 @@
 import boto3
 import botocore
 
+CERT_SUMMARY_LIST_KEY = 'CertificateSummaryList'
+CERT_ARN_KEY = 'CertificateArn'
+DOMAIN_NAME_KEY = 'DomainName'
+
 
 class DiscoACM(object):
     """
     A class to manage the Amazon Certificate Service
 
     """
+    WILDCARD_PREFIX = "*."
 
     def __init__(self, connection=None):
         self._acm = connection
 
+    def _in_domain(self, domain, dns_name):
```

Comment:
May be I got the purpose of this internal method wrong, but I would say that something like this is much easier:

```
if domain.startswith(WILDCARD_PREFIX):
    domain = domain[2:]

return domain in dns_name
```